platform/darwin/src/MGLFeature.mm

```diff
@@ -457,7 +457,11 @@ static CLLocationCoordinate2D toLocationCoordinate2D(const mbgl::Point<T> &point
         NSExpression *identifierExpression = [NSExpression expressionForConstantValue:identifier];
         feature.id = [identifierExpression mgl_featureIdentifier];
     }
-    feature.properties = [attributes mgl_propertyMap];
+
+    if (attributes) {
```

Comment:
Thanks for adding this check - I think it's worth leaving a note here why it's needed. Another Obj-C developer may come along and remove assuming it's not necessary.